lib/ansible/module_utils/vmware.py

```diff
@@ -857,7 +861,8 @@ def get_vm(self):
         user_desired_path = None
 
         if self.params['uuid']:
-            vm_obj = find_vm_by_id(self.content, vm_id=self.params['uuid'], vm_id_type="uuid")
+            vm_obj = find_vm_by_id(self.content, vm_id=self.params['uuid'], vm_id_type="uuid",
```

Comment:
You need to check whether `self.params['uuid_type']`actually exists, if a module forgets to implement it, you'll get exceptions